app/assets/stylesheets/custom.scss

```diff
@@ -4,7 +4,11 @@
 
 /* mixins, variables, etc. */
 /* universal */
-
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing:         border-box;
```

Comment:
Trước và sau các toán tử, dấu hai chấm, sau dấu phẩy và dấu chấm phẩy, để 1 khoảng trắng